src/lib/color-picker/color-picker.service.ts

```diff
@@ -23,18 +23,16 @@ export class MccColorPickerService {
 
   /**
    * Add new color to used colors
-   * @param color string
+   * @param colorString string
    */
-  addColor(color: string): void {
-    if (!color || !isValidColor(color)) {
+  addColor(colorString: string): void {
+    if (!colorString) {
```

Comment:
The lack of validation is causing an error.
Run the project and take a look at demo app.
The last two colors of ```usedStart``` are invalid colors, but the collections try to render something.

The validation is important so invalid colors aren't added.

Reply:
Done. Thanks for the review.